frontends/web/src/api/account.ts

```diff
@@ -41,9 +41,14 @@ export const getAccounts = (): Promise<IAccount[]> => {
     return apiGet(`accounts`);
 };
 
-export type AccountState = 'accountSynced' | 'accountDisabled' | 'fatalError' | 'offlineMode';
+export interface IStatus {
+    disabled: boolean;
+    synced: boolean;
+    fatalError: boolean;
+    offline: boolean;
+}
```

Comment:
What's nice about `AccountState = 'accountSynced' | 'accountDisabled' | 'fatalError' | 'offlineMode'` is that there can only be 1 state, but if all the permutations are possible then this doesn't work of corse.

So can the app be synced AND offline AND disabled at the same time?

Reply:
It wasn't used like that though, cause the getStatus call returned an array of this, not exactly this.

I agree it might be nicer, but out of scope for now - this is only a small refactor to allow showing errors in the end.